spec/sidekiq/worker_killer_spec.rb

```diff
@@ -24,6 +24,19 @@
         expect(subject).to receive(:request_shutdown)
         subject.call(worker, job, queue){}
       end
+      it "should call garbage collect" do
+        allow(subject).to receive(:request_shutdown)
+        expect(GC).to receive(:start).with(full_mark: true, immediate_sweep: true)
+        subject.call(worker, job, queue){}
+      end
+      context "but gc is false" do
```

Comment:
In our codebase, we tend to prefix `context` elements with `when`.

```suggestion
      context "when gc is false" do
```

Reply:
Ok, I will fix it.